nvim-client-0.0.1-20.rockspec

```diff
@@ -12,7 +12,8 @@ dependencies = {
   'lua ~> 5.1',
   'lua-messagepack',
   'luv',
-  'coxpcall'
+  'coxpcall',
+  'luaposix',
```

Comment:
Yeah, i was thinking the same, that dep looked a bit artificial since it  was only used in the test. The `--cmd` alternative looks better.